api/src/indigo_molecule.cpp

```diff
@@ -2665,11 +2664,11 @@ CEXPORT int indigoAddSuperatom(int molecule, int natoms, int* atoms,
 int idx = mol.sgroups.addSGroup(SGroup::SG_TYPE_SUP);
 Superatom& satom = (Superatom&)mol.sgroups.getSGroup(idx);
 satom.subscript.appendString(name, true);
-if (atoms == 0)
+if (atoms == NULL)
```

Comment:
Since we are targeting C++11, it's better to use `nullptr` instead.

Reply:
Done.